### PR TITLE
Update in links having event and formatting

### PR DESCRIPTION
**Pull request by aditmehta9**
## Description

Update in links having event and formatting

## Type of change

- [x] This change requires a documentation update

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings

### CODE REVIEW

**Comment by aditmehta9** [comment]
@michellezhuogg can you please review add gsoc label

File: docs/tutorials/ad_monetization.md [hunk 1]
```diff
@@ -147,7 +147,7 @@ If you are using Shaka's UI library, we will automatically hook up our ad UI.
 #### Listening To Ad Events
 We unify Server Side and Client Side ad events into our own Shaka ad events and
 objects. which your application can listen to and interact with.
-Check out the [full list of ad events][] for details.
+Check out the {@link shaka.ads.AdManager#event:AdBreakReadyEvent|full list of ad events} for details.
```
Comment (michellezhuogg) [inline docs/tutorials/ad_monetization.md:150]: Please wrap each line with 80 characters for code width. (I use the ruler in my text editor to check that.) Thank you!

File: docs/tutorials/ad_monetization.md [hunk 2]
```diff
@@ -172,14 +172,12 @@ adManager.addEventListener(shaka.ads.AdManager.AD_STARTED, (e) => {
...
 
-[full list of ad events]: https://shaka-player-demo.appspot.com/docs/api/shaka.ads.AdManager.html#.event:AdBreakReadyEvent
-
 #### Accommodating IMA Power Users
 If you have an existing IMA integration you want to plug into Shaka, or you want
 to use more intricate SDK capabilities not exposed through our API, we provide a
 way to do that.
-Listen to the [shaka.ads.AdManager.ImaAdManagerLoadedEvent][] for Client Side
-or the [shaka.ads.AdManager.ImaStreamManagerLoadedEvent][] for Server Side to
+Listen to the {@link shaka.ads.AdManager#event:ImaAdManagerLoadedEvent} for Client Side
```
Comment (michellezhuogg) [inline docs/tutorials/ad_monetization.md:179]: Same here.

**Comment by michellezhuogg** [comment]
Thank you for your contribution!

**Pull request merged**